db-acra-migration/python/requirements/base.txt

```diff
@@ -0,0 +1,3 @@
+SQLAlchemy==1.4.15
+pythemis
```

Comment:
let's freeze pythemis version too because we plan to update it and probably may break some compatibility or change public API